src/twisted/python/_url.py

```diff
@@ -528,9 +528,10 @@ def child(self, *segments):
 
         For example::
 
-            >>> (URL.fromText(u"http://localhost/a/b?x=y")
-                 .child(u"c", u"d").asText())
-            u'http://localhost/a/b/c?x=y'
+            >>> u = URL.fromText(u"http://localhost/a/b?x=y")
+            >>> v = u.child(u"c", u"d")
+            >>> print(v.asText())
+            u'http://localhost/a/b/c/d?x=y'
```

Comment:
@rodrigc If this were sphinx (well, or just regular stdlib doctest but those are dark times) then this error would have (or could have) failed an executable doctest which would have shown that the output didn't match what was executed.

So I was just guessing (well, learning) that apparently then nothing actually executes the documentation examples?